service/src/test/java/bio/terra/workspace/service/resource/controlled/flight/CreateGcsBucketStepTest.java

```diff
@@ -159,4 +173,31 @@ public void testCreatesBucketWithoutAllParameters() throws RetryException, Inter
     assertThat(info.getStorageClass(), is(nullValue()));
     assertThat(info.getLifecycleRules(), empty());
   }
+
+  @Test
+  public void createBucket_invalidBucketName_throwsInvalidReferenceException() {
+    doThrow(new StorageException(400, "bad request"))
+        .when(mockStorageCow).create(bucketInfoCaptor.capture());
+
+    // A bad bucket name that fails to be caught by the WSM validation.
+    final String bucketName = uniqueName("bad-bucket-name");
```

Comment:
out of curiosity, what makes this name bad? it seems valid by the guidelines

Reply:
the name is fine. I'm just trying to come up with a generic name that refers to a bad bucket name that wsm's validator fails to catch. it's all mocked out in this unit test so the naming doesn't really matter.